collector/fixtures/e2e-output.txt

```diff
@@ -242,70 +242,70 @@ node_cpu_guest_seconds_total{cpu="7",mode="user"} 0.09
 node_cpu_package_throttles_total{node="0"} 30
 # HELP node_cpu_seconds_total Seconds the cpus spent in each mode.
 # TYPE node_cpu_seconds_total counter
-node_cpu_seconds_total{cpu="cpu0",mode="idle"} 10870.69
-node_cpu_seconds_total{cpu="cpu0",mode="iowait"} 2.2
-node_cpu_seconds_total{cpu="cpu0",mode="irq"} 0.01
-node_cpu_seconds_total{cpu="cpu0",mode="nice"} 0.19
-node_cpu_seconds_total{cpu="cpu0",mode="softirq"} 34.1
-node_cpu_seconds_total{cpu="cpu0",mode="steal"} 0
-node_cpu_seconds_total{cpu="cpu0",mode="system"} 210.45
-node_cpu_seconds_total{cpu="cpu0",mode="user"} 444.9
-node_cpu_seconds_total{cpu="cpu1",mode="idle"} 11107.87
-node_cpu_seconds_total{cpu="cpu1",mode="iowait"} 5.91
-node_cpu_seconds_total{cpu="cpu1",mode="irq"} 0
-node_cpu_seconds_total{cpu="cpu1",mode="nice"} 0.23
-node_cpu_seconds_total{cpu="cpu1",mode="softirq"} 0.46
-node_cpu_seconds_total{cpu="cpu1",mode="steal"} 0
-node_cpu_seconds_total{cpu="cpu1",mode="system"} 164.74
-node_cpu_seconds_total{cpu="cpu1",mode="user"} 478.69
-node_cpu_seconds_total{cpu="cpu2",mode="idle"} 11123.21
-node_cpu_seconds_total{cpu="cpu2",mode="iowait"} 4.41
-node_cpu_seconds_total{cpu="cpu2",mode="irq"} 0
-node_cpu_seconds_total{cpu="cpu2",mode="nice"} 0.36
-node_cpu_seconds_total{cpu="cpu2",mode="softirq"} 3.26
-node_cpu_seconds_total{cpu="cpu2",mode="steal"} 0
-node_cpu_seconds_total{cpu="cpu2",mode="system"} 159.16
-node_cpu_seconds_total{cpu="cpu2",mode="user"} 465.04
-node_cpu_seconds_total{cpu="cpu3",mode="idle"} 11132.3
-node_cpu_seconds_total{cpu="cpu3",mode="iowait"} 5.33
-node_cpu_seconds_total{cpu="cpu3",mode="irq"} 0
-node_cpu_seconds_total{cpu="cpu3",mode="nice"} 1.02
-node_cpu_seconds_total{cpu="cpu3",mode="softirq"} 0.6
-node_cpu_seconds_total{cpu="cpu3",mode="steal"} 0
-node_cpu_seconds_total{cpu="cpu3",mode="system"} 156.83
-node_cpu_seconds_total{cpu="cpu3",mode="user"} 470.54
-node_cpu_seconds_total{cpu="cpu4",mode="idle"} 11403.21
-node_cpu_seconds_total{cpu="cpu4",mode="iowait"} 2.17
-node_cpu_seconds_total{cpu="cpu4",mode="irq"} 0
-node_cpu_seconds_total{cpu="cpu4",mode="nice"} 0.25
-node_cpu_seconds_total{cpu="cpu4",mode="softirq"} 0.08
-node_cpu_seconds_total{cpu="cpu4",mode="steal"} 0
-node_cpu_seconds_total{cpu="cpu4",mode="system"} 107.76
-node_cpu_seconds_total{cpu="cpu4",mode="user"} 284.13
-node_cpu_seconds_total{cpu="cpu5",mode="idle"} 11362.7
-node_cpu_seconds_total{cpu="cpu5",mode="iowait"} 6.72
```

Comment:
this seems like an unnecessary break, seeing that the majority of collectors was using the `cpuX` form before why change it?

Reply:
We're breaking everything in this release, and I wanted to cleanup this unnecessary string duplication.  We already used the new convention for `node_cpu_guest_seconds_total`.

I guess I also need to fix a couple other instances of this old convention.

Reply:
The freebsd collector was the only one to follow the "new" convention, I think it's the most correct.

Reply:
Ah okay, if we're already breaking everything already anyway, then why not!